packages/app/obojobo-document-engine/src/scripts/oboeditor/components/page-editor.js

```diff
@@ -173,6 +179,91 @@ class PageEditor extends React.Component {
 			event.preventDefault()
 			this.editor.redo()
 		}
+
+		if (event.key === 'Escape') {
+			event.preventDefault()
+			ReactEditor.blur(this.editor)
+		}
+
+		// Open top insert menu
+		if (event.key === '-' && (event.ctrlKey || event.metaKey)) {
+			event.preventDefault()
+			// Prevent keyboard stealing by locking the editor to readonly
+			this.editor.toggleEditable(false)
+
+			// Get the first, leafmost Obojobo node
+			// This allows for things to be inserted inside of nested nodes like Questions
+			const [nodeEntry] = Editor.nodes(this.editor, {
+				at: Range.start(this.editor.selection),
+				match: n => Element.isElement(n) && !n.subtype,
+				mode: 'lowest'
+			})
+
+			// Change the node so that the top insert menu is open
+			Transforms.setNodes(
+				this.editor,
+				{ open: 'top' },
+				{
+					at: nodeEntry[1]
+				}
+			)
+		}
+
+		// Open bottom insert menu
+		if (event.key === '=' && (event.ctrlKey || event.metaKey)) {
+			event.preventDefault()
+			// Prevent keyboard stealing by locking the editor to readonly
+			this.editor.toggleEditable(false)
+
+			// Get the first, leafmost Obojobo node
+			// This allows for things to be inserted inside of nested nodes like Questions
+			const [nodeEntry] = Editor.nodes(this.editor, {
+				at: Range.end(this.editor.selection),
+				match: n => Element.isElement(n) && !n.subtype,
+				mode: 'lowest',
+				reverse: true
+			})
+
+			// Change the node so that the top insert menu is open
+			Transforms.setNodes(
+				this.editor,
+				{ open: 'bottom' },
+				{
+					at: nodeEntry[1]
+				}
+			)
+		}
+
+		// Open top insert menu
+		if (event.key === 'i' && (event.ctrlKey || event.metaKey) && event.shiftKey) {
```

Comment:
I can't get this to work because I think when shift is held down event.key is `I`, not `i`.